guides/doc-Project_API/topics/proc_deleting-openscap-reports.adoc

```diff
@@ -0,0 +1,146 @@
+
+include::snip_note-python3-vs-python.adoc[]
+
+.Deleting an OpenSCAP report
+
+To delete an OpenSCAP report, complete the following steps:
+
+. List all OpenSCAP reports.
+Note the IDs of the reports that you want to delete.
++
+Example request:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+curl --insecure --user _username_:_password_ \
+https://_{foreman-example-com}_/api/v2/compliance/arf_reports/ | python3 -m json.tool
+----
++
+Example response:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  3252    0  3252    0     0   4319      0 --:--:-- --:--:-- --:--:--  4318
+{
+    "page": 1,
+    "per_page": 20,
+    "results": [
+        {
+            "created_at": "2017-05-16 13:27:09 UTC",
+            "failed": 0,
+            "host": "_host1.example.com_",
+            "id": 404,
+            "othered": 0,
+            "passed": 0,
+            "updated_at": "2017-05-16 13:27:09 UTC"
+        },
+        {
+            "created_at": "2017-05-16 13:26:07 UTC",
+            "failed": 0,
+            "host": "_host2.example.com_,
+            "id": 405,
+            "othered": 0,
+            "passed": 0,
+            "updated_at": "2017-05-16 13:26:07 UTC"
+        },
+        {
+            "created_at": "2017-05-16 13:25:07 UTC",
+            "failed": 0,
+            "host": "_host3.example.com_",
+            "id": 406,
+            "othered": 0,
+            "passed": 0,
+            "updated_at": "2017-05-16 13:25:07 UTC"
+        },
+        {
+            "created_at": "2017-05-16 13:24:07 UTC",
+            "failed": 0,
+            "host": "_host4.example.com_",
+            "id": 407,
+            "othered": 0,
+            "passed": 0,
+            "updated_at": "2017-05-16 13:24:07 UTC"
+        },
+    ],
+    "search": null,
+    "sort": {
+        "by": null,
+        "order": null
+    },
+    "subtotal": 29,
+    "total": 29
+----
+. Using an ID from the previous step, delete the OpenSCAP report.
+Repeat for each ID that you want to delete.
++
+Example request:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# curl --insecure --user _username_:_password_ \
+--header "Content-Type: application/json" \
+--request DELETE https://_{foreman-example-com}_/api/v2/compliance/arf_reports/405
+----
++
+Example response:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+HTTP/1.1 200 OK
+Date: Thu, 18 May 2017 07:14:36 GMT
+Server: Apache/2.4.6 (Red Hat Enterprise Linux)
+X-Frame-Options: SAMEORIGIN
+X-XSS-Protection: 1; mode=block
+X-Content-Type-Options: nosniff
+Foreman_version: 1.11.0.76
+Foreman_api_version: 2
+Apipie-Checksum: 2d39dc59aed19120d2359f7515e10d76
+Cache-Control: max-age=0, private, must-revalidate
+X-Request-Id: f47eb877-35c7-41fe-b866-34274b56c506
+X-Runtime: 0.661831
+X-Powered-By: Phusion Passenger 4.0.18
```

Comment:
unrelated: I believe we now use Puma as "Ruby HTTP server": https://community.theforeman.org/t/drop-passenger-support/22787

Reply:
It's an example response, is it really that important? I'd have to get a new response and replace it. Another PR, perhaps?